lib/spack/spack/environment.py

```diff
@@ -1560,21 +1470,18 @@ def uninstalled_specs(self):
                     uninstalled_specs.append(spec)
         return uninstalled_specs
 
-    def install_all(self, args=None, **install_args):
+    def install_all(self, **install_args):
         """Install all concretized specs in an environment.
 
         Note: this does not regenerate the views for the environment;
         that needs to be done separately with a call to write().
 
         Args:
-            args (argparse.Namespace): argparse namespace with command arguments
             install_args (dict): keyword install arguments
         """
-        self.install_specs(None, args=args, **install_args)
-
-    def install_specs(self, specs=None, args=None, **install_args):
-        from spack.installer import PackageInstaller
+        self.install_specs(None, **install_args)
 
+    def install_specs(self, specs=None, **install_args):
```

Comment:
Definitely a better API imo without a dependency on an `argparse` namespace. Thanks for taking care of that!